src/lib/features/client-feature-toggles/delta/client-feature-toggle-delta.ts

```diff
@@ -294,4 +305,20 @@ export class ClientFeatureToggleDelta {
     private async updateSegments(): Promise<void> {
         this.segments = await this.segmentReadModel.getActiveForClient();
     }
+
+    storeFootprint() {
+        try {
+            const featuresMemory = this.getCacheSizeInBytes(this.delta);
```

Comment:
How many of these do we keep at the same time? Help me wrap my head around this: can the delta be different for each connected client (assuming they're polling and you're making a flag change every second)?

presumably this uses the revision id to store something, right, so it doesn't need that. I ... remember you saying something about that.

Reply:
We keep a delta for each environment. So all clients that connect from same environment, share it.